cli/runtime/plugin/doc.go

```diff
@@ -36,11 +43,8 @@ var genDocsCmd = &cobra.Command{
 		}
 		emptyStr := func(s string) string { return "" }
 
-		tanzuCmd := cobra.Command{
-			Use: "tanzu",
-		}
 		// Necessary to generate correct output
-		tanzuCmd.AddCommand(cmd.Parent())
+		core.RootCmd.AddCommand(cmd.Parent())
```

Comment:
Looks like incorrect merge conflict resolution as it adds `core` as a dependency.